widgetssdk/src/main/java/com/glia/widgets/view/VisitorCodeView.kt

```diff
@@ -0,0 +1,170 @@
+package com.glia.widgets.view
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.os.Handler
+import android.os.Looper
+import android.util.AttributeSet
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.ProgressBar
+import android.widget.TextView
+import androidx.appcompat.widget.AppCompatImageButton
+import androidx.core.content.ContextCompat
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.content.withStyledAttributes
+import com.glia.androidsdk.omnibrowse.VisitorCode
+import com.glia.widgets.R
+import com.glia.widgets.UiTheme
+import com.glia.widgets.callvisualizer.VisitorCodeContract
+import com.glia.widgets.core.callvisualizer.domain.VisitorCodeRepository
+import com.glia.widgets.di.Dependencies
+import com.glia.widgets.helper.Logger
+import com.glia.widgets.helper.Utils
+import com.glia.widgets.view.unifiedui.exstensions.applyLayerTheme
+import com.glia.widgets.view.unifiedui.exstensions.applyTextTheme
+import com.glia.widgets.view.unifiedui.exstensions.layoutInflater
+import com.google.android.material.theme.overlay.MaterialThemeOverlay
+
+class VisitorCodeView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    closable: Boolean
```

Comment:
This must have a default value to make possible provide all necessary constructors in java, but this is a bad practice anyway for the view I think. Ideally, we can't prevent use of this view inside xml, and it would be better to provide closable as a xml attribute, this way Integrator can use this view wheneverand how it wanted.